### PR TITLE
[MISC] Sync from Discourse

### PR DESCRIPTION
**Pull request by deusebio**
## Issue
The documentation on Discourse is out of sync.
The automated Discourse->GitHub sync system is still WIP, so we need to sync docs on manual request.

## Solution
The sync tool created by @deusebio is able to create PR from Discourse to GitHub,
but it requires manual post-production to avoid mermaid diagrams removal from GitHub.
we have to update automated PRs manually until we are waiting for mermaid support on Discource:
https://github.com/canonical/charmhub.io/issues/1525

### CODE REVIEW

**Review by taurus-forever — CHANGES_REQUESTED** [review]
We cannot merge it AS IS.
The problem in removing "mermaid" diagram. All the rest is OK. @deusebio we need to discuss possible solutions here.

File: docs/explanation/e-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Interfaces/endpoints
+
+Charmed MySQL K8s supports modern `mysql_client` and legacy `mysql` interfaces (in a backward compatible mode).
+
+**Note:** do NOT relate both modern and legacy interfaces simultaneously!
+
+## Modern interfaces
+
+This charm provides modern ['mysql_client' ](https://github.com/canonical/charm-relation-interfaces) interface. Applications can easily connect MySQL using ['data_interfaces'](https://charmhub.io/data-platform-libs/libraries/data_interfaces) library from ['data-platform-libs'](https://github.com/canonical/data-platform-libs/).
```
Comment (shayancanonical) [inline docs/explanation/e-interfaces.md:9]: extra space in `['mysql_client' ]`

**Review by shayancanonical — APPROVED** [review]

**Review by paulomach — APPROVED** [review]
noice

File: docs/explanation/e-interfaces.md (hunk 1 above)
Comment (taurus-forever) [reply docs/explanation/e-interfaces.md:9]: Fixed on Discourse, it will be synced here the next PR.

**Pull request merged**